src/tribler/core/components/knowledge/restapi/knowledge_endpoint.py

```diff
@@ -65,24 +65,21 @@ async def update_knowledge_entries(self, request):
             return error_response
 
         # Validate whether the size of the tag is within the allowed range and filter out duplicate tags.
-        tags = set()
```

Comment:
This `set()` is unnecessary here because statements anyway will be converted to sets later.